modules/documents/client/views/document-manager-edit.html

```diff
@@ -120,12 +120,43 @@ <h3 ng-if="doc._schemaName === 'Folder'" class="modal-title">Edit Folder  &nbsp;
 					</div>
 				</div>
 			</div>
+			<div class="row" ng-if="doc.inspectionType === 'Inspection Report Response'">
+				<div class="form-group datepicker col-md-3" x-show-errors>
+					<label class="control-label" for="dateResponse">Inspection Report Response Date</label>
+					<div class="datepicker-input">
+						<input class="form-control" placeholder="YYYY-MM-DD"  id="dateResponse" name="dateResponse"
+									 ng-model="doc.inspectionReport.dateResponse" type="text"
+									 uib-datepicker-popup
+									 is-open="dateResponsePicker.opened"
+									 datepicker-options="dateOptions"
+									 close-text="Close"
+									 disabled/>
+						<button class="btn btn-default calendar-btn" ng-click="dateResponseOpen()"><span class="glyphicon glyphicon-calendar"/></button>
```

Comment:
The new code followed the pattern that was in the code before. It's hard here in Github to see what you are asking for because I can only see 4 lines of the whole file.